### DISCUSSION TITLE
More information about the user in their profile

### DISCUSSION THREAD

**Discussion by julkasalsa** [RetroAchievements/RAWeb#686] [Ideas]
None of these should be mandatory, but every user should be allowed to bring information about their:
- country of origin (and/or language they are using);
- gender;
- day and month of birth (we could add a feature, to make everyone who have a bithday more visible on the site);
- info about profiles on other sites;

Again this is an option for everyone who is willing to bring additional info like this. It shouldn't be default in registration or especially forced by any meaning.

**Comment by kdecks** [comment]
An about me section would also be nice, a character limited space to write whatever the player pleases.

Also a way to link their favorite games and/or achievements. Could make use of [game=123] forum syntax in the about me or have it's own widget for listing favorites.

---
I think it's a good idea to have all these hidden if you're not logged in, (also the user wall).
